src/frontend/src/btc/services/btc-send.services.ts

```diff
@@ -0,0 +1,37 @@
+import type { UtxosFee } from '$btc/types/btc-send';
+import { convertNumberToSatoshis } from '$btc/utils/btc-send.utils';
+import { selectUserUtxosFee } from '$lib/api/backend.api';
+import type { BtcAddress } from '$lib/types/address';
+import { mapToSignerBitcoinNetwork } from '$lib/utils/network.utils';
+import type { Identity } from '@dfinity/agent';
+import type { BitcoinNetwork } from '@dfinity/ckbtc';
+
+const DEFAULT_MIN_CONFIRMATIONS = 6;
+
+export const selectUtxosFee = async ({
+	identity,
+	sourceAddress,
+	network,
+	amount
+}: {
+	identity: Identity;
+	network: BitcoinNetwork;
+	sourceAddress: BtcAddress;
+	amount: number;
+}): Promise<UtxosFee> => {
+	const satoshisAmount = convertNumberToSatoshis({ amount });
+	const signerBitcoinNetwork = mapToSignerBitcoinNetwork({ network });
+
+	const { fee_satoshis, utxos } = await selectUserUtxosFee({
```

Comment:
You are going to handle try catch and toast on the UI side? Won't implement this here?

Reply:
Good point, better to handle it here, I'll create a separate PR.

Reply:
I just saw that e.g. in case of IC we handle it on the UI side. So, to be consistent, will do the same for BTC.

Reply:
Makes sense. Thanks for double checking.